### PR TITLE
Updates to the Standard Terms of Service - Spring 2018

### PR DESCRIPTION
**Pull request by bluemazzoo**
Most of the changes to our terms are clarifications of pre-existing sections. Here are a few sections we'd like to highlight:

* Third Party Applications: We combined the Marketplace section with general requirements for those creating integrations for other users to provide better protections for GitHub users and their data. The Marketplace section is now called "Third Party Applications," since it now applies to more than just GitHub's Marketplace. We've also added a "Third Party Applications" section to the Privacy Statement to discuss our users' privacy expectations in regards to those applications
* Access to Private Repositories: In Section E, we clarified the purposes for which we may be required to access private repository contents, in line with the security obligations of our GDPR compliance program
* More definitions: We included definitions of "User Accounts" and "Organizations" and described who has control of those types of accounts

### CODE REVIEW

File: Policies/github-terms-of-service.md [hunk 1]
```diff
@@ -43,32 +43,37 @@ Effective date: October 11, 2017
...
-
-#### 4. Additional Terms
+**Short version:** *User Accounts and Organizations have different administrative controls; a human must create your Account; you must be 13 or over; you must provide a valid email address; and you may not have more than one free Account. You alone are responsible for your Account and anything that happens while you are signed in to or using your Account. You are responsible for keeping your Account secure.*
+
+#### 1. Account Controls
+- Users. Subject to these Terms, you retain ultimate administrative control over your User Account and the Content within it.
+
+- Organizations. The "owner" of an Organization that was created under these Terms has ultimate administrative control over that Organization and the Content within it. Within the Service, an owner can manage User access to the Organization’s data and projects. An Organization may have multiple owners, but there must be at least one User Account designated as an owner of an Organization. If you are the owner of an Organization under these Terms, we consider you responsible for the actions that are performed on or through that Organization.
```
[the remark this answers is not in the document]
Comment (bluemazzoo) [reply Policies/github-terms-of-service.md]: Hi @KOLANICH, thank you for your feedback. The purpose of this section is to clarify what Organizations are and how users may control and maintain them. At this time, we will not be making any changes to this section.

**Comment by bluemazzoo** [comment]
Thank you all for your feedback and comments, but we won't be making any additional changes at this time.

**Comment by bluemazzoo** [comment]
The public comment period has ended. This version is being merged into master.

**Comment by sebialex** [comment]
> When access is required for security reasons, including when access is required to maintain ongoing confidentiality, integrity, availability and resilience of GitHub's systems and Service.

So pretty much any time.

**Comment by angelmuazzem** [comment]
I am new one for the hola. Let me know about this.

**Comment by angelmuazzem** [comment]
After a little more, I will be a donnate.

**Pull request merged**